src/app/lib/config/services/arabic.js

```diff
@@ -169,6 +168,11 @@ export const service = {
       numberOfItems: 10,
       hasMostRead: true,
     },
+    radioSchedule: {
+      hasRadioSchedule: true,
+      frequenciesPageUrl:
+        '/arabic/institutional/2011/01/000000_frequencies_radio',
```

Comment:
I think it would be easier if we include `https://www.bbc.com/` so that we don't have to append this in the future.

Reply:
Thanks @thekp . However, I think they're available on all of the different environments (`https://www.bbc.com/` & `https://www.bbc.co.uk/`). So it might be best to keep them as they are

Reply:
Ah okay, that’s fine then.